services/antivirus/pkg/service/service.go

```diff
@@ -142,7 +142,7 @@ func (av Antivirus) Run() error {
 		}
 
 		av.l.Info().Str("uploadid", ev.UploadID).Interface("resourceID", ev.ResourceID).Str("virus", res.Description).Str("outcome", string(outcome)).Str("filename", ev.Filename).Str("user", ev.ExecutingUser.GetId().GetOpaqueId()).Bool("infected", res.Infected).Msg("File scanned")
-		if err := events.Publish(stream, events.PostprocessingStepFinished{
+		if err := events.Publish(context.Background(), stream, events.PostprocessingStepFinished{
```

Comment:
is there a reason why the context from line 107 is not reused?

Reply:
Mostly because I was going by a checklist of problems, and was kinda in auto pilot mode by then. Good catch. Fixed now.